packages/plugin-loyalties-ui/src/loyalties/coupons/components/CampaignList.tsx

```
@@ -0,0 +1,93 @@
+    router.removeParams(navigate, location, 'campaignId');
+  };
+
+  const isActive = (id: string) => {
+    const currentGroup = queryParams?.campaignId || '';
+
+    return currentGroup === id;
+  };
+
+  const renderContent = () => {
+    const otherParams = { ...queryParams };
+    delete otherParams.campaignId;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)